modules/sno-du-configuring-the-operators.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-configuring-the-operators_{context}"]
+= Configuring the Operators
+
+This configuration is provided to add the Operators needed to configure the platform post
+installation. It adds the Local storage Operator, Logging Operator, Performance addon Operator,
+PTP Operator and the SRIOV Network Operator.
```

Comment:
This module only describes how to prepare the cluster for installing the operators. The operator installation is done by the subscriptions below.